pages/common/aws-secretsmanager.md

```diff
@@ -0,0 +1,32 @@
+# aws secretsmanager
+
+> Store, manage, and retrieve secrets.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/secretsmanager/>.
+
+- Show secrets stored by the secrets manager in the current account:
+
+`aws secretsmanager list-secrets`
+
+- Create a secret:
+
+`aws secretsmanager create-secret --name {{name}} --description "{{String describing secret}}" --secret-string {{secret}}`
+
+- Delete a secret:
+
+`aws secretsmanager delete-secret --secret-id {{name_or_arn}}`
+
+- View details of a secret except for secret text:
+
+`aws secretsmanager describe-secret --secret-id {{name_or_arn}}`
+
+- Retrieve the value of a secret (to get the latest version of the secret omit `--version-stage`):
+
+`aws secretsmanager get-secret-value --secret-id {{name_or_arn}} --version-stage {{version_of_secret}}`
+
+- Rotate the secret immediately using a Lambda function:
+
+`aws secretsmanager rotate-secret --secret-id {{name_or_arn}} --rotation-lambda-arn {{arn_of_lambda_function}}`
+
+- Rotate the secret automatically every 30 days using a Lambda function:
+
+`aws secretsmanager rotate-secret --secret-id {{name_or_arn}} --rotation-lambda-arn {{arn_of_lambda_function}} --rotation-rules AutomaticallyAfterDays={{30}}`
```

Comment:
Is the argument option `AutomaticallyAfterDays` just one example? I guess there are more posible values.
```suggestion
`aws secretsmanager rotate-secret --secret-id {{name_or_arn}} --rotation-lambda-arn {{arn_of_lambda_function}} --rotation-rules {{AutomaticallyAfterDays=30}}`
```

Reply:
Yeah, but since the example is explicitly about automatically rotating, I think it's better to keep just the `30` in token syntax.

Reply:
I can't find any other argument option in the documentation.